src/utils/padNumber.js

```diff
@@ -0,0 +1,5 @@
+function padNumber(value) {
```

Comment:
we are using `arrow function (=>)` and `function` syntax to define utils functions, I'd recommend to keep it consistent using just `arrow functions (=>)` or `function` syntax.